style.css

```
@@ -0,0 +1,133 @@
+:root {
+  --font-family-heading: "Anton", sans-self;
+  --font-family-boby: "Noto Sans JP", sans-serif;
+
+  --color-white: #fffefc;
+  --color-black: #020911;
+  --color-accent-yellow: #f2e52e;
+  --color-accent-red: #f20574;
+  --color-background-black: #171525;
+  --color-background-navy: #030427;
+  --color-form-background-yellow: #fffab9;
+  --color-form-background-gray: #ab9783;
+  --color-form-boder-red: #df6da2;
+  --color-button-hover-red: #f043a5;
+  --color-course-hover-yellow: #f4e877;
+  --color-hero-overlay: rgba(57, 24, 111, 0.4);
+  --color-course-overlay: rgba(32, 8, 19, 0.63);
+}
+
+/* global */
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: var(--font-family-body);
+}
+
+img {
+  display: block;
+}
+
+/* header */
+.header {
+  height: 80px;
+  background-color: var(--color-background-black);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 10;
+  width: 100%;
+}
+
+.header-logo-wrapper {
+  margin: 0;
+}
+
+.header-logo {
+  height: 50px;
+}
+
+@media (max-width: 750px) {
+  .header {
+    height: 60px;
+  }
+  .header-logo {
+    height: 30px;
+  }
+}
+
+/*hero*/
+.hero {
+  background-image: url("./img/hero.png");
+  background-size: cover;
+  display: flex;
+  justify-content: center;
+  padding: 94px 40px;
```

```suggestion
  padding: 96px 40px;
```
すいません。動画上ではpadding: 94px 40px;となっているかと思うのですがfigmaが作成するデザインでは
padding: 96px 40px;　となっているので修正お願いします